[EV-4240] Remove references to values.yaml replacement (#1028)
tigera/docs: calico-enterprise/multicluster/helm/create-a-managed-cluster-helm.mdx

  export MANAGED_CLUSTER_KEY=<managed-cluster-key>
  ```

> 1. Verify the managed cluster context has been correctly updated in the `values.yaml`.

Reviewer: ```suggestion
1. Verify that the managed cluster context is correctly updated in the `values.yaml`.
```
